internal/osutils/autostart/autostart.go

```diff
@@ -1,50 +1,78 @@
 package autostart
 
 import (
+	"github.com/ActiveState/cli/internal/constants"
 	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/locale"
 )
 
 const ConfigKeyDisabled = "SystrayAutoStartDisabled"
 
-type App struct {
-	Name string
-	Exec string
-	cfg  Configurable
+type AppName string
+
+func (a AppName) String() string {
+	return string(a)
+}
+
+const (
+	Tray    AppName = constants.TrayAppName
+	Service         = constants.SvcAppName
```

Comment:
We should move this out of autostart. This libs function should be all about facilitating autostarts, not about implementing specific versions of things that want to autostart. So svc and tray specific stuff should live in places that own that aspect, eg. cmd/state-tray and cmd/state-svc.

This config key would be another candidate for this.